lib/couchbase-orm/persistence.rb

```diff
@@ -201,9 +204,9 @@ def reload
             raise "unable to reload, model not persisted" unless key
 
             CouchbaseOrm.logger.debug "Data - Get #{key}"
-            resp = self.class.bucket.get(key, quiet: false, extended: true)
-            @__attributes__ = ::ActiveSupport::HashWithIndifferentAccess.new(resp.value)
-            @__metadata__.key = resp.key
+            resp = self.class.collection.get!(key)
```

Comment:
if get fail, we should to raise custom couchbase-orm error

Reply:
must to have